scipy/special/add_newdocs.py

```diff
@@ -8784,34 +8783,33 @@ def add_newdoc(name, doc):
     >>> from scipy.stats import norm
     >>> n = 5
     >>> gendist = norm(0.5, 1)       # Normal distribution, mean 0.5, stddev 1
-    >>> np.random.seed(seed=233423)  # Set the seed for reproducibility
     >>> x = np.sort(gendist.rvs(size=n))
     >>> x
-    array([-0.20946287,  0.71688765,  0.95164151,  1.44590852,  3.08880533])
+    array([-0.20946287,  0.71688765,  0.95164151,  1.44590852,  3.08880533])  # random
     >>> target = norm(0, 1)
     >>> cdfs = target.cdf(x)
     >>> cdfs
-    array([ 0.41704346,  0.76327829,  0.82936059,  0.92589857,  0.99899518])
+    array([ 0.41704346,  0.76327829,  0.82936059,  0.92589857,  0.99899518])  # random
     # Construct the Empirical CDF and the K-S statistics (Dn+, Dn-, Dn)
     >>> ecdfs = np.arange(n+1, dtype=float)/n
     >>> cols = np.column_stack([x, ecdfs[1:], cdfs, cdfs - ecdfs[:n], ecdfs[1:] - cdfs])
     >>> np.set_printoptions(precision=3)
     >>> cols
-    array([[ -2.095e-01,   2.000e-01,   4.170e-01,   4.170e-01,  -2.170e-01],
+    array([[ -2.095e-01,   2.000e-01,   4.170e-01,   4.170e-01,  -2.170e-01],  # random
            [  7.169e-01,   4.000e-01,   7.633e-01,   5.633e-01,  -3.633e-01],
            [  9.516e-01,   6.000e-01,   8.294e-01,   4.294e-01,  -2.294e-01],
            [  1.446e+00,   8.000e-01,   9.259e-01,   3.259e-01,  -1.259e-01],
            [  3.089e+00,   1.000e+00,   9.990e-01,   1.990e-01,   1.005e-03]])
     >>> gaps = cols[:, -2:]
     >>> Dnpm = np.max(gaps, axis=0)
     >>> print('Dn-=%f, Dn+=%f' % (Dnpm[0], Dnpm[1]))
-    Dn-=0.563278, Dn+=0.001005
+    Dn-=0.563278, Dn+=0.001005  # random
     >>> probs = smirnov(n, Dnpm)
     >>> print(chr(10).join(['For a sample of size %d drawn from a N(0, 1) distribution:' % n,
-    ...      ' Smirnov n=%d: Prob(Dn- >= %f) = %.4f' % (n, Dnpm[0], probs[0]),
+    ...      ' Smirnov n=%d: Prob(Dn- >= %f) = %.4f' % (n, Dnpm[0], probs[0]),  # random
```

Comment:
False positive? I don't think we need this here.

Reply:
Thanks I will check!